packages/components/RadioGroup/src/RadioGroup.types.ts

```diff
@@ -34,30 +34,36 @@ export interface IRadioGroupState {
 }
 
 export interface IRadioGroupProps {
-  /*
-   ** Descriptive label for the RadioGroup. This will be displayed as the title of the radio group to the user
+  /**
+   * Descriptive label for the RadioGroup. This will be displayed as the title of the radio group to the user.
    */
   label: string;
 
-  /*
-   ** The key of the RadioButton that will initially be selected
+  /**
+   * The key of the RadioButton that will initially be selected
    */
   defaultSelectedKey?: string;
 
-  /*
-   **  OPTIONAL: An aria label for narrator. If not provided, it will be set to the label of the radio group
+  /**
+   *  A label for screen readers. If not provided, it will be set to the label of the radio group.
+   * @deprecated Use accessibilityLabel instead.
    */
   ariaLabel?: string;
 
   /*
-   ** The key of the selected option. If you provide this, you must maintain selection state by observing
-   ** onChange events and passing a new value in when changed. This overrides defaultSelectedKey
-   ** and makes the RadioGroup a controlled component.
+   ** An accessibility label for screen readers. If not provided, it will be set to the label of the radio group.
+   */
+   accessibilityLabel?: string;
```

Comment:
I added in `accessibilityLabel` as a prop here but alternatively it could be added if we had IRadioGroupProps extend IViewProps. @samuelfreiberg do you have any thoughts here? Should we add IViewProps to RadioGroup?

Reply:
I'll probably check it in as is for now and we can update it later if we decide to change it.